app/routes/studio_assets.py

```
@@ -534,6 +536,99 @@ async def reclassify_asset(
+            config = CKOConfig()
+
+            async for event in cko_service.stream_message(
+                session_id=session.id,
+                user_id=user_id,
+                message=asset_context,
+                config=config
+            ):
+                event_type = event.get("type", "unknown")
+                yield f"event: {event_type}\ndata: {json.dumps(event)}\n\n"
+
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)